config/schema/3.0/patches/db_schema_patch-3.1.sql

```diff
@@ -164,4 +164,28 @@ VALUES("Occurrence Associations Type", "omoccurassociations", "relationship", "a
 INSERT INTO ctcontrolvocabterm(cvID, term, termDisplay)
 SELECT cvID, "fieldNotes", "Field Notes" FROM ctcontrolvocab WHERE tableName = "omoccurassociations" AND fieldName = "relationship" AND filterVariable = "associationType:resource";
 INSERT INTO ctcontrolvocabterm(cvID, term, termDisplay)
-SELECT cvID, "genericResource", "Generic Resource" FROM ctcontrolvocab WHERE tableName = "omoccurassociations" AND fieldName = "relationship" AND filterVariable = "associationType:resource";
\ No newline at end of file
+SELECT cvID, "genericResource", "Generic Resource" FROM ctcontrolvocab WHERE tableName = "omoccurassociations" AND fieldName = "relationship" AND filterVariable = "associationType:resource";
+
+# Establish a table to track third party auth
+
+CREATE TABLE `users_thirdpartyauth` (
```

Comment:
Fixed in 314f89508.

Regarding, 
> And we already have a table called useraccesstokens. Should we just modify that, rather than creating a new table that serves the same function?

I'm open either way. I know that we still have some more work to do with this table in terms of log out post 3.1. Perhaps let's kick the potentialy to fuse those tables to post 3.1?

Reply:
And my bad - that is clearly spelled out in the coding conventions doc haha.